instrumentation/elasticsearch/elasticsearch-rest-6.4/javaagent/elasticsearch-rest-6.4-javaagent.gradle

```diff
@@ -23,12 +23,12 @@ testSets {
 dependencies {
   compileOnly group: 'org.elasticsearch.client', name: 'elasticsearch-rest-client', version: '6.4.0'
 
-  implementation project(':instrumentation:elasticsearch:elasticsearch-rest-common')
+  implementation project(':instrumentation:elasticsearch:elasticsearch-rest-common:javaagent')
 
-  testImplementation project(':instrumentation:apache-httpclient:apache-httpclient-4.0')
-  testImplementation project(':instrumentation:apache-httpasyncclient-4.0')
+  testImplementation project(':instrumentation:apache-httpclient:apache-httpclient-4.0:javaagent')
+  testImplementation project(':instrumentation:apache-httpasyncclient-4.0:javaagent')
   // Netty is used, but it adds complexity to the tests since we're using embedded ES.
```

Comment:
```suggestion
  //TODO: review the following claim, we are not using embedded ES anymore
  // Netty is used, but it adds complexity to the tests since we're using embedded ES.
```